cypress/e2e/Testcases/TC2_Logout.spec.cy.js

```diff
@@ -0,0 +1,23 @@
+import loginPage from "../Pages/LoginPage";
+import LogoutPage from "../Pages/LogoutPage";
+
+describe('Logout scenario', function () {
+  it('should log in and then log out', function () {
+    // Visit the Sauce Demo website
+    loginPage.visit();
+
+    // Log in with valid credentials
+    loginPage.setUsername();
+    loginPage.setPassword();
+    loginPage.clickLoginButton();
+
+    //Wait for the menu to become visible
+    LogoutPage.MenuVisibility();
```

Comment:
This method doesn't wait for menu visibility. Please consider refactoring to do what is expected. :)